examples/miscellaneous/plot_outlier_detection_bench.py

```diff
@@ -186,8 +186,14 @@ def compute_prediction(X, model_name):
             name=model_name,
             linewidth=linewidth,
             ax=axs[i // cols, i % cols],
+            plot_chance_level=True,
```

Comment:
I assume that you only want to plot for the latest call. So you can use the same trick:

```python
for model_idx, model_name in models_name:
    plot_chance_level = model_idx == (len(models_name) - 1)
    ...
```